### DISCUSSION TITLE
Player can still access the interface of handheld devices when operating a periscope

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#13984] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

you can see custom interface of handheld devices like handheld sonar, handheld status monitor when using a periscope
it does not make sense since you are looking through the periscope

### Reproduction steps

1. Equip a handheld sonar
2. Operate a periscope
3. Sonar interface is still there

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
![20240522003415_1](https://github.com/FakeFishGames/Barotrauma/assets/64390906/02ee8878-d6d8-4d72-91d7-58496a8ba062)

**Comment by Regalis11** [comment]
While this is a little strange, it's intentional, there can be some uses for accessing the device while using the periscope.
